pyexcel_ods3/odsr.py

```diff
@@ -43,7 +43,10 @@ def cell_value(self, row, column):
             if service.has_no_digits_in_float(cell_value):
                 cell_value = int(cell_value)
 
-            ret = str(cell_value) + " " + cell.currency
+            if cell.currency is None:
+                ret = str(cell_value)
+            else:
+                ret = str(cell_value) + " " + cell.currency
```

Comment:
```suggestion
            ret = str(cell_value) if cell.currency is None else " ".join(str(cell_value), cell.currency)
```

Reply:
With the formatter applied, IMO, it becomes less readable:

```
            ret = (
                str(cell_value)
                if cell.currency is None
                else " ".join([str(cell_value), cell.currency])
            )
```